CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0] - 2020-02-28
+
+### Changed
+
+Update to the latest version of protocol buffers from rippled, introduced in [#3254](https://github.com/ripple/rippled/pull/3254).
```

Comment:
What about this is a breaking change? Can you give a little more context?

I know you can add new fields to Protobufs in a backwards-compatible way, so maybe explain what the breaking change in `rippled` was, and how that breaks the API exposed by `xpring-common-js`.